packages/suite-base/src/panels/StateTransitions/openSiblingStateTransitionsPanel.test.ts

```diff
@@ -0,0 +1,47 @@
+// SPDX-FileCopyrightText: Copyright (C) 2023-2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)<lichtblick@bmwgroup.com>
+// SPDX-License-Identifier: MPL-2.0
+
+import { StateTransitionConfig } from "@lichtblick/suite-base/panels/StateTransitions/types";
+import BasicBuilder from "@lichtblick/suite-base/testing/builders/BasicBuilder";
+import { OpenSiblingPanel } from "@lichtblick/suite-base/types/panels";
+
+import { openSiblingStateTransitionsPanel } from "./openSiblingStateTransitionsPanel";
+
+describe("openSiblingStateTransitionsPanel", () => {
+  let mockOpenSiblingPanel: OpenSiblingPanel;
+  let topicName: string;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    mockOpenSiblingPanel = jest.fn();
+    topicName = BasicBuilder.string();
+  });
+
+  it("should call openSiblingPanel with correct parameters", () => {
+    openSiblingStateTransitionsPanel(mockOpenSiblingPanel, topicName);
+    expect(mockOpenSiblingPanel).toHaveBeenCalledWith({
+      panelType: "StateTransitions",
+      updateIfExists: true,
+      siblingConfigCreator: expect.any(Function),
+    });
+
+    const config: StateTransitionConfig = { paths: [], isSynced: false };
```

Comment:
@luluiz 

I've tried to change it, but since the modification on config is being done after the call of the method, those constants are part of the "when".

Reply:
Waiting for your suggestions/feedback to resolve this conversation

Reply:
Ok, sometimes this kind of junction happens.